pkg/autodiscover/autodiscover_operators.go

```diff
@@ -34,14 +35,9 @@ const (
 	istioNamespace = "istio-system"
 )
 
-func findIstioNamespace(oc corev1client.CoreV1Interface) bool {
-	nsList, err := oc.Namespaces().List(context.TODO(), metav1.ListOptions{})
-	if err != nil {
-		logrus.Errorln("Error when listing", "err: ", err)
-		return false
-	}
-	for index := range nsList.Items {
-		if nsList.Items[index].ObjectMeta.Name == istioNamespace {
+func findIstioNamespace(allNs *[]string) bool {
```

Comment:
Slices are already pass-by-reference.  
```suggestion
func findIstioNamespace(allNs []string) bool {
```

Reply:
fixed